uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/resources/org/uberfire/client/views/PatternFly.gwt.xml

```diff
@@ -30,4 +30,5 @@
   <stylesheet src="bootstrap-select/css/bootstrap-select.min.css"/>
   <stylesheet src="prettify/bin/prettify.min.css"/>
   <stylesheet src="uberfire-patternfly.css" />
+  <stylesheet src="monaco-editor/dev/vs/editor/editor.main.css"/>
```

Comment:
If you decide to switch to `min` remember to change this path too!

Reply:
Fixed! It's fine to use the minified CSS :-)